__tests__/app.test.js

```diff
@@ -174,4 +174,24 @@ describe("NC News App", () => {
         });
     });
   });
+
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("Status: 200 and replies with an array of comments objects", () => {
+      return request(app)
+        .get("/api/articles/3/comments")
+        .expect(200)
+       .then(({ body }) => {
+          expect(body.comments).toHaveLength(2);
+          body.comments.forEach((comment) => {
+          expect(comment).toEqual(expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            created_at: expect.any(String)
+            }));
```

Comment:
We need more testing here, think about;
- what if we request an articles comment that doesnt exist?
- an article_id that isnt a number
- the article exists but has no comments?